REQUIREMENTS.txt

```diff
@@ -1,2 +1,3 @@
+scipy
```

Comment:
I believe scipy is already in QGIS, right?, so a user can still run InaSAFE without installing scipy.